okhttp/src/main/java/io/grpc/okhttp/OkHttpClientStream.java

```diff
@@ -101,6 +101,11 @@
             transport,
             initialWindowSize,
             method.getFullMethodName());
+
+    // TODO(ejona86) Fix a potential data race using MigratingThreadDeframer in OkHttp.
+    // OkHttp is lock based and it doesn't get benefit from using MigratingThreadDeframer,
```

Comment:
It did actually see a benefit. So let's just say we're disabling it for the race.